ios/fastlane/Fastfile

```diff
@@ -210,10 +210,8 @@ platform :ios do
     # See https://github.com/fastlane/fastlane/issues/20741
     ENV['ITMSTRANSPORTER_FORCE_ITMS_PACKAGE_UPLOAD'] = 'false'
 
-    # Setup the circleci environment
-    setup_ci(
-      provider: "circleci"
-    )
+    # Setup the CI environment
+    setup_ci()
```

Comment:
Why do we fallback to provider automatic detection? Cannot we pass the new provider name (e.g `"github-actions"`)?

Reply:
looking at the doc there's no specific provider for github-actions ref: https://docs.fastlane.tools/actions/setup_ci/

Reply:
That's wired 🥶

Reply:
Okay. [That's the reason why](https://github.com/fastlane/fastlane/blob/7b3e224acc72df66dcfca785ca94d7f6cc4ae36f/fastlane/lib/fastlane/actions/setup_ci.rb#L10) ;)